site/content/en/docs/Reference/fleetautoscaler.md

```diff
@@ -177,6 +177,125 @@ spec:
 See the [Webhook Endpoint Specification](#webhook-endpoint-specification) for the specification of the incoming and 
 outgoing JSON packet structure for the webhook endpoint.
 
```

Comment:
I think we might need to add a block like the one mentioned here https://agones.dev/site/docs/contribute/#within-a-page so that it renders in the version we want.  v1.43.0 (?)

Reply:
I see, since it hasn't been realesed yet, sg!

Reply:
It will be rolled out in 1.43.0 which gets released tomorrow.